<launch>
  <node pkg="base_controller" type="docking_safe_node" name="docking_safe_node" output="screen">
    <param name="safe_dist" value="0.27" type="double"/>
    <param name="kp" value="0.8" type="double"/>
    <param name="ki" value="1.0" type="double"/>
    <param name="kd" value="0.0" type="double"/>
    <param name="output_limit" value="0.3" type="double"/>
    <param name="integral_limit" value="0.3" type="double"/>
    <remap from="yocs_cmd_vel_mux/input/safe" to="yocs_cmd_vel_mux/input/remote"/>
  </node>

</launch>
